src/DbSql/TaoRdf/UnionQuerySerialyser.php

```diff
@@ -179,7 +179,7 @@ public function addOperator($expression) {
             $this->query .=  $this->getDriverEscaper()->dbCommand('AND') . ' '.
                 $this->getDriverEscaper()->reserved('modelid') . ' '.
                 $this->getDriverEscaper()->dbCommand('IN') . ' '.
-                '(' . implode(',', $this->model->getReadableModels()) . ')'.
+                '(' . implode(',', array_map(function($a) { return "'" .$a . "'"; }, $this->model->getReadableModels())) . ')'.
```

Comment:
what about `"('" . implode("','", $this->model->getReadableModels() . "')"` for readability ?

Reply:
$this->model->getReadableModels() shouldn't be empty, since at least userland model is created.
And even if it was empty, it would generate `AND modelid IN ('')`, which couldn't hurt at all...